build_scripts/windows/scripts/build.cmd

```diff
@@ -193,14 +207,19 @@ for /d %%d in ("azure*.dist-info") do (
 )
 popd
 
-if %errorlevel% neq 0 goto ERROR
 
-echo Building MSI...
-msbuild /t:rebuild /p:Configuration=Release /p:Platform=%ARCH% %REPO_ROOT%\build_scripts\windows\azure-cli.wixproj
+if "%TARGET%"=="msi" (
+    echo Building MSI...
+    msbuild /t:rebuild /p:Configuration=Release /p:Platform=%ARCH% %REPO_ROOT%\build_scripts\windows\azure-cli.wixproj
+) else (
+    echo Building ZIP...
+    ren %BUILDING_DIR% "Azure CLI"
```

Comment:
Use `Azure CLI` as root folder of zip package.
PS: The MSI installation path is `C:\Program Files\Microsoft SDKs\Azure\CLI2`.